test/globalInitialization.ts

```diff
@@ -13,7 +13,7 @@ before(() => {
   if (window.PHANTOMJS) {
     window.Pixel_CloseTo_Requirement = 2;
   } else if (isFirefox) {
-    window.Pixel_CloseTo_Requirement = 1;
+    window.Pixel_CloseTo_Requirement = 2;
```

Comment:
Can you add the explanation that you had about `getBBox()` and `getBoundingClientRect()` to the comments on the source here and annotate it with `HACKHACK:`? That way this explanation won't be lost for the future.